<script>
    fetch('http://localhost:9999/todos')
        .then((response)=>{
            if(!response.ok){
                throw new Error('Network response was not ok');
            }
            return response.json()
        })
        .then((data)=>{
            console.log(data);
        })
        .catch((error)=>{
            console.error('Fetch error:', error);
        });

        const getTodos = async ()=>{
            try{
                const response = await fetch('http://localhost:9999/todos');
                if(!response.ok){
                    throw new Error('Network response was not ok');
                }

                const data = await response.json();
                console.log(data);
            } catch(error){
                console.error('Fetch error:',error);
            }
        };
        getTodos();
</script>
